tahoe-server/storage0/tahoe.cfg

```
@@ -28,9 +28,9 @@ helper.furl =
 # This can be changed at any time: the encoding is saved in
 # each filecap, and we can download old files with any encoding
 # settings
-shares.needed = 3
-shares.happy = 7
-shares.total = 10
+shares.needed = 1
```

We can rever to the original default because it turns out that this does not affect behavior.
```suggestion
shares.needed = 3
shares.happy = 7
shares.total = 10
```